src/Features/Missions/missionsSlice.js

```diff
@@ -0,0 +1,40 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+export const getMissionsAsync = createAsyncThunk(
+  'missions/getMissionsAsync',
+  async () => {
+    const response = await fetch('https://api.spacexdata.com/v3/missions');
+    if (response.ok) {
+      const data = await response.json();
+      const missions = data;
+      return missions;
+    }
+    return null;
+  },
+);
```

Comment:
- Great way to fetch API but It will be better to find a way to handle errors on it by using rejectWithValue from thunkAPI 👍 refers to this [OPTIONAL]
![image](https://user-images.githubusercontent.com/51321655/233045442-b55c5a15-766d-4cba-a4f4-842d73e72d23.png)